packages/ckeditor5-ui/src/button/buttonlabelview.ts

```diff
@@ -0,0 +1,66 @@
+/**
+ * @license Copyright (c) 2003-2023, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/**
+ * @module ui/button/buttonlabelview
+ */
+
+import View from '../view';
+import type ButtonLabel from './buttonlabel';
+
+/**
+ * A default implementation of the button view's label. It comes with a dynamic text support
+ * via {@link module:ui/button/buttonlabelview~ButtonLabelView#text} property.
+ */
+export default class ButtonLabelView extends View implements ButtonLabel {
+	/**
+	 * @inheritDoc
+	 */
+	declare public id: string | undefined;
+
+	/**
+	 * @inheritDoc
+	 */
+	declare public style: string | undefined;
+
+	/**
+	 * @inheritDoc
+	 */
+	declare public text: string | undefined;
+
+	/**
+	 * @inheritDoc
+	 */
+	constructor() {
+		super();
+
+		this.set( {
+			style: undefined,
```

Comment:
Do we need to set those values to `undefined`? I think it is the default behavior.

Reply:
It occurs in several places

Reply:
These properties are observable and have to be initialized this way for DOM bindings to be correct.

Reply:
Right.